support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -17,7 +17,6 @@ export const pageUrlRegexes = {
   subscriptions: {
     subsShowcaseAndDigiSubPages: '(/??/subscribe(\\?.*)?$|/??/subscribe/digital(\\?.*)?$)',
     digiSubLandingPages: '(/??/subscribe/digital/gift(\\?.*)?$|/??/subscribe/digital(\\?.*)?$)',
-    digiSubLandingPagesNotAus: '(/(uk|us|ca|eu|nz|int)/subscribe/digital(\\?.*)?$)',
```

Comment:
I often think it's worth leaving these in for the next time someone wants to use it. Don't mind too much either way though.

Reply:
Yes, great point. Thanks Rupert